manifests/params.pp

```diff
@@ -33,6 +33,35 @@
         ],
         'maxconn' => '8000'
       }
+        $config_file      = '/etc/haproxy/haproxy.cfg'
+      }
+    'FreeBSD': {
+      $package_name     = 'haproxy'
+      $global_options   = {
+        'log'           => [
+          '127.0.0.1 local0',
+          '127.0.0.1 local1 notice',
+        ],
+        'chroot'        => '/usr/share/haproxy',
```

Comment:
I agree that changing the chroot default to /usr/local/haproxy is a good
idea.  I'll make that change.  Were you suggesting that the log defaults
change as well?

On Fri, Feb 13, 2015, 22:33 Zach Leslie <notifications@github.com> wrote:

> In manifests/params.pp
> <https://github.com/puppetlabs/puppetlabs-haproxy/pull/148#discussion_r24711723>
> :
>
> > @@ -33,6 +33,35 @@
> >          ],
> >          'maxconn' => '8000'
> >        }
> > +        $config_file      = '/etc/haproxy/haproxy.cfg'
> > +      }
> > +    'FreeBSD': {
> > +      $package_name     = 'haproxy'
> > +      $global_options   = {
> > +        'log'           => [
> > +          '127.0.0.1 local0',
> > +          '127.0.0.1 local1 notice',
> > +        ],
> > +        'chroot'        => '/usr/share/haproxy',
>
> I was only thinking that if the port doesn't patch that file (which it
> doesn't look to) then those are likely just the defaults for haproxy and
> there may be a more appropriate place on FreeBSD to locate those.
> /usr/local/haproxy for example might fit better to the FreeBSD way of doing
> things.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-haproxy/pull/148/files#r24711723>
> .
>